pylint/test/input/func_e0303.py

```diff
@@ -0,0 +1,44 @@
+"""Check bad value returned by __len__ """
```

Comment:
My initial attempt was to write them there, although I had some issues with running them. Should they be discovered via `test_func.py` or via discovered unittests?